GitCommands/Git/GitModule.cs

```diff
@@ -431,7 +431,15 @@ public bool HasSubmodules()
         public IReadOnlyList<string> GetSubmodulesLocalPaths(bool recursive = true)
         {
             var localPaths = new List<string>();
-            DoGetSubmodulesLocalPaths(this, "", localPaths, recursive);
+            try
+            {
+                DoGetSubmodulesLocalPaths(this, "", localPaths, recursive);
+            }
+            catch (GitConfigurationException)
```

Comment:
@RussKie 

After making what seems to be the finishing touches on this PR (other than squashing), I've decided to interactively test this.  I've found that upon opening a git repo with a malformed config file, the exceptions thrown by the parser are lost here in this catch block.  The comment claims that any config exceptions would have already been shown to the user at this point, but I have not seen any such error messages.

Reply:
👍 
We've iterated few time over the code, so it is likely this is no longer correct.
What happens you remove both instances of `try..catch` from this class?

Reply:
@RussKie 

When I remove both of the `try..catch` blocks from the class, I get this non-fatal pop-up upon opening GE and upon hitting refresh:

![on-start-RELEASE](https://user-images.githubusercontent.com/14078392/72114902-1baded80-3313-11ea-8688-e3b4fe912f7c.PNG)

When I go to the "Diff" tab on a commit, I get this fatal bug report pop-up:

![on-diff-tab-RELEASE](https://user-images.githubusercontent.com/14078392/72114904-1cdf1a80-3313-11ea-874a-1edac028b943.PNG)


The same behavior occurs if I ONLY remove the `try..catch` block in `GitCommands.GitModule.GetSubmodulesLocalPaths`

If I remove only the `try..catch` from `GitCommands.GitModule.GetSubmodulesInfo`, I do not get these pop-up windows.  However, if I go to commit 853f1cc2e0dec946128e7f36fb91e1f2c95987e4 (a commit that has changes in a submodule) and go to the "Diff" tab, I see this little message in the changed-files list:
![image](https://user-images.githubusercontent.com/14078392/72114796-b5c16600-3312-11ea-98a4-366a03ff812a.png)

Doing a search of "bad config" in the entire GitExtensions.sln solution, I cannot find this string at all and suspect this changed-files list is a third-party component that GE utilizes.

Reply:
I'll have a look